src/main/kotlin/com/hibob/academy/employeeFeedback/dao/FeedbackDao.kt

```diff
@@ -83,4 +74,29 @@ class FeedbackDao(private val sql: DSLContext) {
             .and(feedbackTable.companyId.eq(companyId))
             .execute()
     }
+
+    fun viewAllSubmittedFeedback(filterRequest: FeedbackFilterBy, companyId: Int): List<FeedbackInfo> {
+        val query = sql.select(feedbackTable.id, feedbackTable.employeeId,
+                                feedbackTable.companyId, feedbackTable.dateOfFeedback,
+                                feedbackTable.anonymity, feedbackTable.reviewed,
+                                feedbackTable.feedback)
+                        .from(feedbackTable)
+                        .leftJoin(employeeTable)
+                        .on(employeeTable.id.eq(feedbackTable.employeeId))
```

Comment:
why do you need the join? you are not querying anything from the table

Reply:
I would do the join only if a department filter was received because I don't want to always query from that table.

You can fix it later, if you have time